API/api.py

```diff
@@ -318,6 +341,19 @@ def updateData(event_code: str):
                 CalculatedDataCollection.update_one({"event_code": YEAR+event_code}, {'$set': {"data": data, "metadata": metadata}})
             except Exception as ex:
                 pass
+
```

Comment:
What are the activate and deactivate methods for? Does this determine if a team is actively competing? If so shouldn't the API know this data internally? No rest endpoint should be required.